yang/frr-interface.yang

```diff
@@ -43,4 +47,12 @@ module frr-interface {
       }
     }
   }
+
+  typedef interface-ref {
+    type leafref {
+      path "/frr-interface:lib/frr-interface:interface/frr-interface:name";
+    }
```

Comment:
Shouldn't `require-instance false;` be used here as we discussed today?

Without that, we can have this kind of problem:
```
debian(config)# router rip
debian(config-router)# offset-list ACL in 1 eth999
debian(config-router)# commit
% Failed to commit candidate configuration: validation error.

Please check the logs for more details.
```
```
RIP: libyang: Invalid value "eth999" in "interface" element. (/frr-ripd:ripd/instance[vrf='default']/offset-list[interface='eth999'][direction='in']/interface)                                                          
RIP: [EC 100663337] nb_candidate_commit_prepare: failed to validate candidate configuration
```

Most of our configuration commands allow the user to refer to objects that don't yet exist, so we need to preserve this behavior. When necessary, the `require-instance` setting can be overwritten for commands that need to refer to existing objects.

Reply:
I rebased and amended that thing so many times today I forgot to add that line 😁 
You're correct.